.github/workflows/claude-review-from-author.yml

```
@@ -0,0 +1,41 @@
+          direct_prompt: |
+            Please provide a thorough review of this pull request.
+
+            Since this is from a specific author that requires careful review,
+            please pay extra attention to:
+            - Adherence to project coding standards
+            - Proper error handling
+            - Security best practices
+            - Test coverage
+            - Documentation
+
+            Provide detailed feedback and suggestions for improvement.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)